cms/djangoapps/contentstore/views/tests/test_course_marketing_site.py

```diff
@@ -0,0 +1,73 @@
+"""
+Unit tests for course settings UI in Studio with marketing site flags.
+"""
+
+import mock
+
+from contentstore.tests.utils import CourseTestCase
+from contentstore.utils import reverse_course_url
+from student.roles import CourseInstructorRole, CourseStaffRole, GlobalStaff
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.factories import CourseFactory
+
+class CourseMarketingSiteTest(CourseTestCase):
+    """Base class to test course settings details view in Studio when using marketing site flag.
+    """
+    def setUp(self):
+        super(CourseMarketingSiteTest, self).setUp()
+        self.course = CourseFactory.create(org='edX', number='dummy', display_name='Marketing Site Course')
+        self.course_details_url = reverse_course_url('settings_handler', unicode(self.course.id))
+
+    @mock.patch.dict("django.conf.settings.FEATURES", {'ENABLE_MKTG_SITE': False})
+    def test_course_details_with_disabled_setting_global_staff(self):
+        """Test that user enrollment end date is not readonly in response
+        if the feature flag 'ENABLE_MKTG_SITE' is not enabled for global staff..
+        """
+        user = UserFactory()
+        GlobalStaff().add_users(user)
+        self.client.login(username=user.username, password='test')
+
+        response = self.client.get_html(self.course_details_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, "Contact your edX Program Manager to update these settings.")
```

Comment:
I believe the new title for PMs is "Partner Manager": 
"Contact your edX Partner Manager to update these settings."
@explorerleslie @bderusha